dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.lra-1.0.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.org.eclipse.microprofile.lra-1.0
+singleton=true
+-bundles=io.openliberty.org.eclipse.microprofile.lra.1.0; location:="dev/api/stable/,lib/"; mavenCoordinates="org.eclipse.microprofile.lra:microprofile-lra-api:1.0.0.M1"
```

Comment:
I think the maven version should be just `1.0-M1`
https://repo1.maven.org/maven2/org/eclipse/microprofile/lra/microprofile-lra-api/1.0-M1/microprofile-lra-api-1.0-M1.jar

... but you could change that in a follow up PR

Reply:
In the bnd file, this has to have a dot, as it doesn't seem to be able to handle the dash. I think I also copied this from another .feature file which did similar.
I'm also not sure what the purpose of that attribute is.

Reply:
Yes, I know about the bnd issue ... but this is a feature file. I *think* that property is used to generate a pom file for the API jar.